### DISCUSSION TITLE
Ongoing content recache works

### DISCUSSION THREAD

**Discussion by arantzaetxebarria** [INSPIRE-MIF/helpdesk-registry#62] [Announcements]
Dear community:

Due to the new release 2023.02 and the INSPIRE Registry's content re-factoring work, some content may take some time to be updated.

Sorry for the inconvenience

**Comment by KathiSchleidt** [comment]
@arantzaetxebarria any outlook when this content re-factoring work will be completed? 

I just checked the [WRB Qualifier Values](https://inspire.ec.europa.eu/codelist/WRBQualifierValue) codelist, interesting is that while the main codelist looks empty, there are actually values if one creates ones own URL, e.g. [Entic](https://inspire.ec.europa.eu/codelist/WRBQualifierValue/Entic), [Protothionic](https://inspire.ec.europa.eu/codelist/WRBQualifierValue/Protothionic).
I don't quite understand why refactoring is required here, as the alignment of the WRB Qualifier Value entries with the WRB Qualifier Values codelist were clear from the import documents.

In addition, we see the same error with the other codelists provided under #33. Below a listing of these seemingly empty codelists, together with a hidden entry: 
- [FAOHorizonMasterValue](https://inspire.ec.europa.eu/codelist/FAOHorizonMasterValue/): [H](http://inspire.ec.europa.eu/codelist/FAOHorizonMasterValue/H) 
- [FAOHorizonSubordinateValue](https://inspire.ec.europa.eu/codelist/FAOHorizonSubordinateValue): [a](https://inspire.ec.europa.eu/codelist/FAOHorizonSubordinateValue/a)
- [FAOPrimeValue](https://inspire.ec.europa.eu/codelist/FAOPrimeValue/): [0](https://inspire.ec.europa.eu/codelist/FAOPrimeValue/0)

We've been waiting for over a year to have these values imported, would be thankful to be able to finally close #33!

**Comment by arantzaetxebarria** [comment]
Dear @KathiSchleidt 

As you've rightly pointed out, the new items introduced are visible in the application. Due to problems, we have detected related to the cache, the items are not visible in the main codelist tables.

We are working on it and analysing the root cause of this situation and why this massive publication has not been fully cached.

Regards

**Comment by arantzaetxebarria** [comment]
I would like to inform you that the problems related to [WRB Qualifier Values](https://inspire.ec.europa.eu/codelist/WRBQualifierValue) have been solved. Now the table with available items is visible

Regards
